static/app/actionCreators/formSearch.tsx

```diff
@@ -30,9 +30,9 @@ const createSearchMap = ({
   return listOfFields.map<FormSearchField>(field => ({
     ...other,
     route,
-    title: typeof field !== 'function' ? (field.label as string) : undefined,
-    description: typeof field !== 'function' ? (field.help as string) : undefined,
-    field,
+    title: typeof field !== 'function' ? (field?.label as string) : undefined,
+    description: typeof field !== 'function' ? (field?.help as string) : undefined,
+    field: field as Field,
```

Comment:
idk if hardcoding this is right

Reply:
you’re right, I inspected the type error a bit more, and it’s basically “just” an `undefined is not assignable to type FieldObject` error. Using a non-null assertion works, too, and is likely the better approach here. Fixed it here:

b34d108c